tests/scripts/stubs/hostname

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
```

Comment:
please use env here to determine the shell.

Reply:
this file needs a copyright header.

Reply:
Done